### PR TITLE
add readme for how to connect cloudsql via cloudsql proxy

### PR DESCRIPTION
**Pull request by yonghaoy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,19 @@ You can now push to the specified environment by running
...
 ```
 
+## Connecting psql client using the Cloud SQL Proxy:
+Go to cloud console to get the instance name you want to connect, then start the proxy:
+```
+./cloud_sql_proxy -instances=<INSTANCE_CONNECTION_NAME>=tcp:5432
+```
+Start the client session
+```
+psql "host=127.0.0.1 sslmode=disable dbname=<DB_NAME> user=<USER_NAME>"
```
Comment (wchamber) [inline README.md:62]: Can you add how a Broad person might find the right instance name/db name/user name (no password needed?).

File: README.md [hunk 2]
```diff
@@ -52,6 +52,19 @@ You can now push to the specified environment by running
 skaffold run
 ```
 
+## Connecting psql client using the Cloud SQL Proxy:
+Go to cloud console to get the instance name you want to connect, then start the proxy:
+```
+./cloud_sql_proxy -instances=<INSTANCE_CONNECTION_NAME>=tcp:5432
```
Comment (wchamber) [inline README.md:58]: I don't see a ./cloud_sql_proxy file in the PR.

File: README.md [hunk 3]
```diff
@@ -52,6 +52,19 @@ You can now push to the specified environment by running
 skaffold run
 ```
 
+## Connecting psql client using the Cloud SQL Proxy:
+Go to cloud console to get the instance name you want to connect, then start the proxy:
```
Comment (wchamber) [inline README.md:56]: english nit: "want to connect to, then.."

File: README.md [hunk 4]
```diff
@@ -52,6 +52,19 @@ You can now push to the specified environment by running
...
+```
+./cloud_sql_proxy -instances=<INSTANCE_CONNECTION_NAME>=tcp:5432
+```
+Start the client session
+```
+psql "host=127.0.0.1 sslmode=disable dbname=<DB_NAME> user=<USER_NAME>"
+```
+
+Note that to stop local postgres to free the 5432 port first.
```
Comment (wchamber) [inline README.md:65]: english nit: Note that you must stop the local postgres first to free the 5432 port.

File: README.md (hunk 2 above)
Comment (yonghaoy) [reply README.md:58]: Added

File: README.md (hunk 1 above)
Comment (yonghaoy) [reply README.md:62]: Done

**Comment by yonghaoy** [comment]
@wchamber Hi Will, sorry, I totally forgot this PR. Comments are addressed, PTAL, thanks!

File: README.md [hunk 5]
```diff
@@ -61,6 +61,26 @@ You can now push to the specified environment by running
...
+
+Go to cloud console to get the instance name you want to connect to, then start the proxy:
+```
+./cloud_sql_proxy -instances=<INSTANCE_CONNECTION_NAME>=tcp:5432
+```
+Start the client session
+```
+psql "host=127.0.0.1 sslmode=disable dbname=<DB_NAME> user=<USER_NAME>"
+```
+For Broad engineer, DB_NAME and USER_NAME can be found in vault.
```
Comment (wchamber) [inline README.md:76]: add that the instance name can be found under `...crl_janitor/postgres/instance` in vault?

Comment (yonghaoy) [reply README.md:76]: done

**Pull request merged**